tests/tests_ut/az_ulib_ipc_ut/az_ulib_ipc_ut.c

```diff
@@ -52,6 +52,7 @@ static AZ_ULIB_RESULT set_my_property(const void* const model_in) {
 typedef struct my_method_model_in_tag {
   uint8_t action;
   const az_ulib_interface_descriptor* descriptor;
+  uint32_t wait_police_ms;
```

Comment:
>wait_police_ms [](start = 11, length = 14)

wait_policy?

Reply:
Yes, thanks!

---
In reply to: [342861430](https://github.com/Azure/azure-ulib-c/pull/47#discussion_r342861430) [](ancestors = 342861430)